packages/rooks/src/hooks/useHighlight.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * useHighlight
+ * @description search for a keyword in the DOM and highlight them
+ * @see {@link https://rooks.vercel.app/docs/useHighlight}
+ */
+import { useEffect, useRef } from "react";
+
+function useHighlight({
```

Comment:
Super clever. I have a small preference for argument structure. Can we do something such as this. 

```tsx
useHighlight(keyword, {
   when: true
})
```

That way it will match some of the other hooks in our library. What do you think?

Reply:
Yes, I modify it for the consistency